src/applications/vaos/covid-19-vaccine/index.jsx

```diff
@@ -33,20 +32,22 @@ import {
   selectCanUseVaccineFlow,
   selectDirectScheduleSettingsStatus,
 } from '../appointment-list/redux/selectors';
+import { fetchDirectScheduleSettings } from '../appointment-list/redux/actions';
 
-export function NewBookingSection({
-  canUseVaccineFlow,
-  featureCovid19Vaccine,
-  directScheduleSettingsStatus,
-  fetchDirectScheduleSettings,
-}) {
+export default function NewBookingSection() {
```

Comment:
This wasn't a default export before and the tests are importing the function assuming it's not the default.

Reply:
Ok yeah I see what's wrong now.